src/styles/holographic.css

```
@@ -1,856 +1,157 @@
+  overflow: hidden;
 }
 
-.comm-indicator.speaking .wave-line {
-  animation: voiceWaves 0.5s ease-in-out infinite alternate;
+.holographic-title {
+  background: linear-gradient(90deg, #64b5f6, #ba68c8);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-weight: 800;
+  letter-spacing: 1px;
+  text-shadow: 0 0 20px rgba(100, 181, 246, 0.4);
```

Stylelint (17.2.0)

[error] Expected "0.4" to be "40%" (alpha-value-notation)

(alpha-value-notation)